src/test/resources/workspace/samplewithModelineLike.yaml

```diff
@@ -0,0 +1,13 @@
+// camel-k: language=yaml
```

Comment:
yaml comments are using # and not //
see https://yaml.org/spec/1.2/spec.html#id2780069
modeline is supporting both notation. See https://vim.fandom.com/wiki/Modeline_magic for several example  notations

with //, the yaml file is invalid (can be tested online on http://www.yamllint.com/ )

Reply:
good catch. Will fix that.

Reply:
regarding to the spec comments start with #<space>. Will implement it that way.